.github/workflows/main.yaml

```diff
@@ -0,0 +1,209 @@
+name: CSharp(C#) Client CI Run
```

Comment:
Could we use similar workflow name with others? example: Singleton CSharp(C#)-Client CI

Reply:
CI title is updated.